metadata/src/main/java/org/apache/kafka/metadata/authorizer/trie/package-info.java

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Maybe change the filename to `PackageInfo.java` and also what's the purpose of this file?